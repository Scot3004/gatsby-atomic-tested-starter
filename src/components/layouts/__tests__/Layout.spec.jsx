import React from 'react'
import 'jest-styled-components'
import renderer from 'react-test-renderer'
import Layout from '..'

test('it works', () => {
  const tree = renderer.create(<Layout title='title_stub' />).toJSON()
  expect(tree).toMatchSnapshot()
})
