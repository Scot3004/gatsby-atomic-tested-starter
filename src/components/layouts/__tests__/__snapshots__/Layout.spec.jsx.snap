// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  margin: 3rem auto;
  max-width: 1000px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100%;
}

<div
  className="c0"
>
  <header>
    <h1>
      title_stub
    </h1>
  </header>
</div>
`;
